bench/Main.hs

```diff
@@ -6,12 +6,13 @@ module Main where
 -------------------------------------------------------------------------------
 
 import Control.Exception qualified as E
-import Control.Monad.Except
+import Control.Monad (unless)
+import Control.Monad.Trans.Except (runExceptT)
```

Comment:
`Control.Monad.Except` re-exports a lot of stuff that it shouldn't; we should avoid importing utilities defined in other packages wherever possible.